morpheus/_lib/src/utilities/http_server.cpp

```diff
@@ -127,18 +123,24 @@ class Session : public std::enable_shared_from_this<Session>
     {
         DLOG(INFO) << "Received request: " << request.method() << " : " << request.target();
         m_response = std::make_unique<http::response<http::string_body>>();
+        bool valid_request = false;
 
-        if (request.target() == m_url_endpoint && (request.method() == m_method))
+        for (const auto& endpoint : m_endpoints)
```

Comment:
I would replace the range loop with an old-style loop allowing the loop to exit early once `valid_request` is `true`

Reply:
Added `break` statement instead